### ISSUE TITLE
Would be kinda' neat if you could eyedrop from surroundings

### ISSUE THREAD

**Issue by Jeiji** [edge-reflow/issues#149]
When in the color picker, it'd be cool if I could just select a color from an element I've already created in the workspace. Though the automatically saved swatches are neat, it's just weird I can't do it. Just a suggestion.

**Comment by vmitnick** [comment]
It's not exactly what you're suggesting, but you _can_ copy and paste visual styles from one element to another. It won't handle the more general case like "use the background of that element as the font color for this element", but if you have one box with a certain shadow, for example, you can copy that shadow style to another box. (It copies all visual styles; you can't pick and choose.)

Thanks for the idea, though!

**Comment by Jeiji** [comment]
Yeah, I did see that, which is pretty neat. I should make shortcut keys for it like I did in PS!
